src/main/js/field/UploadFileButtonField.js

```diff
@@ -185,6 +185,7 @@ class UploadFileButtonField extends Component {
 }
 
 UploadFileButtonField.propTypes = {
+    id: PropTypes.string,
```

Comment:
Is this property used?

Reply:
The usage of `{...this.props}` will set the id on the LabeledField.  Have to confirm the LabeledField is setting it properly.